discord/interactions.py

```diff
@@ -173,6 +184,23 @@ def _from_data(self, data: InteractionPayload):
         self.version: int = data['version']
         self.channel_id: Optional[int] = utils._get_as_snowflake(data, 'channel_id')
```

Comment:
the docs mention we should stop using this, and instead use the id in the `channel` field. Maybe do that too.